examples/02_generate_address.rs

```diff
@@ -5,34 +5,27 @@
 // In this example we will generate an address
 // Rename `.env.example` to `.env` first
 
-use std::{env, path::PathBuf};
+use std::env;
 
 use dotenv::dotenv;
-use iota_wallet::{
-    account_manager::AccountManager,
-    secret::{stronghold::StrongholdSecretManager, SecretManager},
-    Result,
-};
+use iota_wallet::{account_manager::AccountManager, Result};
 
 #[tokio::main]
 async fn main() -> Result<()> {
     // This example uses dotenv, which is not safe for use in production
     dotenv().ok();
-    // Setup Stronghold secret_manager
-    let secret_manager = StrongholdSecretManager::builder()
-        .password(&env::var("STRONGHOLD_PASSWORD").unwrap())
-        .snapshot_path(PathBuf::from("wallet.stronghold"))
-        .build();
 
     // Create the account manager
-    let manager = AccountManager::builder()
-        .with_secret_manager(SecretManager::Stronghold(secret_manager))
-        .finish()
-        .await?;
+    let manager = AccountManager::builder().finish().await?;
 
     // Get the account we generated with `01_create_wallet`
     let account = manager.get_account("Alice").await?;
 
+    // Set the stronghold password
+    manager
+        .set_stronghold_password(&env::var("STRONGHOLD_PASSWORD").unwrap())
+        .await?;
```

Comment:
Why is this not part of the builder ?

Reply:
Because the password can also be cleared and then we need a way to set it again, but we could also add it additionally to the builder